thor/src/lib.rs

```diff
@@ -123,425 +109,14 @@ macro_rules! step_wallet {
     }};
```

Comment:
:wrench: I think the two macros in this file (above) should go in the `channel` module, since they won't ever be used anywhere else, and they only make sense in that context. I would put them either in `channel.rs` or a `/channel/macros.rs` file.

Reply:
Done, thanks.

Reply:
Thanks for the review @lucas. All suggestions implemented. I did a few extra patches on top that were clean up things done before but that slipped back in during redase. Merging without further review since there is nothing additional of interest.